composer.json

```
@@ -25,7 +25,7 @@
         }
     },
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
         "ext-mbstring": "*",
         "laminas/laminas-stdlib": "^3.6.0"
```

```suggestion
        "laminas/laminas-stdlib": "^3.19.0"
```

This will fix _some_ test failures on 8.4 latest. You will need to refresh composer.lock

---

Also run `composer bump -D` to bump _all_ dev dependencies. This should solve more PHP 8.4 compatibility issues.